src/skeleton/types.ts

```diff
@@ -0,0 +1,2 @@
+export type SkeletonAnimation = 'progress' | 'none' | 'pulse';
+export type SkeletonVariant = 'line' | 'rect' | 'circle';
```

Comment:
The design system variants are different. but I think those variants can be computed using these. 
We can go with this state and change it if its not flexible enough.